tests/settings.yaml

```diff
@@ -3,7 +3,7 @@ locations:
   reads-dir: sample_data/reads
   reference-fasta: sample_data/NC_045512.2.fasta
   amplicons-bed: sample_data/nCoV-2019_NCref.bed
-  kraken-db-dir: databases/kraken_db
-  krona-db-dir: databases/krona_db
+  kraken-db-dir: /local/Projects/AAkalin_pathogenomics/databases/kraken_db
+  krona-db-dir: /local/Projects/AAkalin_pathogenomics/databases/krona_db
```

Comment:
please don't do this.  All locations are resolved relative to the settings file, so this shouldn't even have been necessary for testing purposes.